site/_docs/reference.md

```diff
@@ -1205,6 +1205,7 @@ Note:
 | MULTISET | Unordered collection that may contain duplicates | Example: int multiset
 | ARRAY    | Ordered, contiguous collection that may contain duplicates | Example: varchar(10) array
 | CURSOR   | Cursor over the result of executing a query |
+| LAMBDA_EXPRESSION| A function definition that is not bound to an identifier.| Example FUNCTION(INTEGER, VARCHAR(30)) -> INTEGER
```

Comment:
Let's just call this a FUNCTION type. Add spaces around `|`. Remove '.'

You should note that it is not fully supported e.g. in CAST or DDL